code/game/turfs/simulated/floor/transparent.dm

```diff
@@ -44,8 +44,18 @@
 /turf/simulated/floor/transparent/glass/crowbar_act(mob/user, obj/item/I)
 	if(!I || !user)
 		return
-	var/obj/item/stack/R = user.get_inactive_hand()
-	if(istype(R, /obj/item/stack/sheet/metal))
+	var/obj/item/stack/R
+	if(ishuman(user))
+		R = user.get_inactive_hand()
+	if(isrobot(user))
+		var mob/living/silicon/robot/robouser = user
+		if(istype(robouser.module_state_1,/obj/item/stack/sheet/metal))
+			R=robouser.module_state_1
+		if(istype(robouser.module_state_2,/obj/item/stack/sheet/metal))
+			R=robouser.module_state_2
+		if(istype(robouser.module_state_3,/obj/item/stack/sheet/metal))
+			R=robouser.module_state_3
+	if(istype(R,/obj/item/stack/sheet/metal))
```

Comment:
```suggestion
	var/obj/item/stack/R
	if(ishuman(user))
		R = user.get_inactive_hand()
	else if(isrobot(user))
		var/mob/living/silicon/robot/robouser = user
		if(istype(robouser.module_state_1, /obj/item/stack/sheet/metal))
			R = robouser.module_state_1
		else if(istype(robouser.module_state_2, /obj/item/stack/sheet/metal))
			R = robouser.module_state_2
		else if(istype(robouser.module_state_3, /obj/item/stack/sheet/metal))
			R = robouser.module_state_3

	if(istype(R, /obj/item/stack/sheet/metal))
```

Reply:
So, I was told that else if's are not a thing that we should use, and was shown this pr: #17204. I'm not sure if that applies or what but I'm just going off what I've been told.